src/test/java/com/cag/cagbackendapi/services/user/UserServiceTest.kt

```diff
@@ -42,10 +42,10 @@ class UserServiceTest {
     }
 
```

Comment:
Don't forget the updateUser UserService unit tests!!

Reply:
added the UserService Unit Tests!